padertorch/contrib/cb/transfrom.py

```diff
@@ -0,0 +1,191 @@
+import typing
```

Comment:
I guess the typo in the filename is not intended (transfrom -> transform)

Reply:
Thanks, fixed.